fhirpath/main.bal

```diff
@@ -0,0 +1,72 @@
+import ballerina/io;
+import ballerinax/health.fhir.r4utils.fhirpath;
+map<json> patient = {
```

Comment:
```suggestion
import ballerinax/health.fhir.r4utils.fhirpath;

map<json> patient = {
```

Reply:
Fixed in d19b2b33bd7c2beff8c27c2c4cb4472342d3d455